src/rpc/rpc.cc

```diff
@@ -339,7 +339,6 @@ DGL_REGISTER_GLOBAL("distributed.rpc._CAPI_DGLRPCHandleSignal")
   sigemptyset(&sigHandler.sa_mask);
   sigHandler.sa_flags = 0;
   sigaction(SIGINT, &sigHandler, nullptr);
-  sigaction(SIGTERM, &sigHandler, nullptr);
```

Comment:
why do you delete this?

Reply:
I added it back, for the shared memory cleaning. But I found catching SIGTERM prevents python to show the correct Exceptions sometime.